libimgui-null-backend-test/manifest

```diff
@@ -0,0 +1,17 @@
+: 1
+name: libimgui-null-backend-test
+version: 1.86.0-a.0.z
+project: imgui
+summary: libimgui-null-backend-test for libimgui
```

Comment:
Optional: You could make the summary a bit more expressive, e.g. `Dear ImGui test package using the null backend`.

Reply:
I would prefer to stay true to what's happening in the real repo: `Dear ImGUI's null backend example used mostly for testing` or something like that.